kqueen_ui/blueprints/ui/views.py

```diff
@@ -618,6 +623,11 @@ class ClusterResize(KQueenView):
 
     def handle(self, cluster_id):
         cluster = self.kqueen_request('cluster', 'get', fnargs=(cluster_id,))
+
```

Comment:
Remove this part pls, I have already PR prepared for improving the message overall and I don't want any conditions checking for specific engine in a code which can by written generically just fine.

Reply:
This is the PR: https://github.com/Mirantis/kqueen-ui/pull/28 - it handles both Jenkins and Manual engines without explicit conditions

Reply:
okok